lib/literal.rb

```diff
@@ -28,6 +28,69 @@ module Literal
 	autoload :TypeError, "literal/errors/type_error"
 	autoload :ArgumentError, "literal/errors/argument_error"
 
+	TRANSFORMS = {
```

Comment:
This mapping allows us to skip type checking array mappings that use a symbol proc since we already know what type it will return.